diff --git a/src/game/recomp/gen/recomp_0057.c b/src/game/recomp/gen/recomp_0057.c
index d7741fe..879385a 100644
--- a/src/game/recomp/gen/recomp_0057.c
+++ b/src/game/recomp/gen/recomp_0057.c
@@ -36,7 +36,7 @@ loc_002BCA78: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D1078;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -72,7 +72,7 @@ loc_002BCA83: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D1078;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -126,7 +126,7 @@ loc_002BCAA8: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D10A4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -215,7 +215,7 @@ loc_002BCAFA: ;
     g_seh_ebp = ebp; sub_00239AF0(); return; /* tail jmp 0x00239AF0 */
 
     eax = 0x3D1150;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -271,7 +271,7 @@ loc_002BCB08: ;
     g_seh_ebp = ebp; sub_00239AF0(); return; /* tail jmp 0x00239AF0 */
 
     eax = 0x3D1150;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -324,7 +324,7 @@ loc_002BCB16: ;
     g_seh_ebp = ebp; sub_00239AF0(); return; /* tail jmp 0x00239AF0 */
 
     eax = 0x3D1150;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -374,7 +374,7 @@ loc_002BCB24: ;
     g_seh_ebp = ebp; sub_00239AF0(); return; /* tail jmp 0x00239AF0 */
 
     eax = 0x3D1150;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -421,7 +421,7 @@ loc_002BCB32: ;
     g_seh_ebp = ebp; sub_00239AF0(); return; /* tail jmp 0x00239AF0 */
 
     eax = 0x3D1150;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -465,7 +465,7 @@ loc_002BCB40: ;
     g_seh_ebp = ebp; sub_00239AF0(); return; /* tail jmp 0x00239AF0 */
 
     eax = 0x3D1150;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -504,7 +504,7 @@ loc_002BCB42: ;
     g_seh_ebp = ebp; sub_00239AF0(); return; /* tail jmp 0x00239AF0 */
 
     eax = 0x3D1150;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -540,7 +540,7 @@ loc_002BCB4D: ;
     g_seh_ebp = ebp; sub_00239AF0(); return; /* tail jmp 0x00239AF0 */
 
     eax = 0x3D1150;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -603,7 +603,7 @@ loc_002BCB78: ;
     g_seh_ebp = ebp; sub_0023B710(); return; /* tail jmp 0x0023B710 */
 
     eax = 0x3D1184;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -636,7 +636,7 @@ loc_002BCB83: ;
     g_seh_ebp = ebp; sub_0023B710(); return; /* tail jmp 0x0023B710 */
 
     eax = 0x3D1184;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -735,7 +735,7 @@ loc_002BCBE8: ;
     g_seh_ebp = ebp; sub_00045F50(); return; /* tail jmp 0x00045F50 */
 
     eax = 0x3D1200;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -768,7 +768,7 @@ loc_002BCBF3: ;
     g_seh_ebp = ebp; sub_00045F50(); return; /* tail jmp 0x00045F50 */
 
     eax = 0x3D1200;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -819,7 +819,7 @@ loc_002BCC18: ;
     g_seh_ebp = ebp; sub_002478C0(); return; /* tail jmp 0x002478C0 */
 
     eax = 0x3D122C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -868,7 +868,7 @@ loc_002BCC38: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D1284;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -911,7 +911,7 @@ loc_002BCC58: ;
     g_seh_ebp = ebp; sub_0023A4B0(); return; /* tail jmp 0x0023A4B0 */
 
     eax = 0x3D12B0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -963,7 +963,7 @@ loc_002BCC78: ;
     g_seh_ebp = ebp; sub_0025DCA0(); return; /* tail jmp 0x0025DCA0 */
 
     eax = 0x3D12DC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1096,7 +1096,7 @@ loc_002BCD3B: ;
     g_seh_ebp = ebp; sub_000E9A70(); return; /* tail jmp 0x000E9A70 */
 
     eax = 0x3D13E0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1265,7 +1265,7 @@ loc_002BCE1B: ;
     g_seh_ebp = ebp; sub_0023A4B0(); return; /* tail jmp 0x0023A4B0 */
 
     eax = 0x3D1558;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1292,7 +1292,7 @@ loc_002BCE26: ;
     g_seh_ebp = ebp; sub_0023A4B0(); return; /* tail jmp 0x0023A4B0 */
 
     eax = 0x3D1558;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1379,7 +1379,7 @@ loc_002BCE88: ;
     g_seh_ebp = ebp; sub_00045F50(); return; /* tail jmp 0x00045F50 */
 
     eax = 0x3D15CC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1469,7 +1469,7 @@ loc_002BCEE8: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D1664;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1795,7 +1795,7 @@ loc_002BD0CB: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D1960;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1895,7 +1895,7 @@ loc_002BD138: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D19F8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2056,13 +2056,13 @@ loc_002BD21B: ;
     g_seh_ebp = ebp; sub_00249E10(); return; /* tail jmp 0x00249E10 */
 
     eax = 0x3D1B68;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     ecx = ebp + -68;
     g_seh_ebp = ebp; sub_000E9A70(); return; /* tail jmp 0x000E9A70 */
 
     eax = 0x3D1B8C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2098,7 +2098,7 @@ loc_002BD230: ;
     g_seh_ebp = ebp; sub_000E9A70(); return; /* tail jmp 0x000E9A70 */
 
     eax = 0x3D1B8C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2194,7 +2194,7 @@ loc_002BD29B: ;
     g_seh_ebp = ebp; sub_0023A4B0(); return; /* tail jmp 0x0023A4B0 */
 
     eax = 0x3D1C10;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2232,7 +2232,7 @@ loc_002BD2A6: ;
     g_seh_ebp = ebp; sub_0023A4B0(); return; /* tail jmp 0x0023A4B0 */
 
     eax = 0x3D1C10;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2267,7 +2267,7 @@ loc_002BD2AE: ;
     g_seh_ebp = ebp; sub_0023A4B0(); return; /* tail jmp 0x0023A4B0 */
 
     eax = 0x3D1C10;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2404,7 +2404,7 @@ loc_002BD35B: ;
     g_seh_ebp = ebp; sub_00257E20(); return; /* tail jmp 0x00257E20 */
 
     eax = 0x3D1CF0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2475,7 +2475,7 @@ loc_002BD38B: ;
     g_seh_ebp = ebp; sub_0025DCA0(); return; /* tail jmp 0x0025DCA0 */
 
     eax = 0x3D1D24;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2518,7 +2518,7 @@ loc_002BD399: ;
     g_seh_ebp = ebp; sub_0025DCA0(); return; /* tail jmp 0x0025DCA0 */
 
     eax = 0x3D1D24;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2576,7 +2576,7 @@ loc_002BD3CB: ;
     g_seh_ebp = ebp; sub_00045F50(); return; /* tail jmp 0x00045F50 */
 
     eax = 0x3D1D58;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2603,7 +2603,7 @@ loc_002BD3D6: ;
     g_seh_ebp = ebp; sub_00045F50(); return; /* tail jmp 0x00045F50 */
 
     eax = 0x3D1D58;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2661,7 +2661,7 @@ loc_002BD404: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D1DA8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2697,7 +2697,7 @@ loc_002BD410: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D1DA8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2836,7 +2836,7 @@ loc_002BD4C4: ;
     g_seh_ebp = ebp; sub_0025AB90(); return; /* tail jmp 0x0025AB90 */
 
     eax = 0x3D1F28;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     ecx = ebp + -80;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
@@ -2856,7 +2856,7 @@ loc_002BD4F9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D1F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2889,7 +2889,7 @@ loc_002BD4C6: ;
     g_seh_ebp = ebp; sub_0025AB90(); return; /* tail jmp 0x0025AB90 */
 
     eax = 0x3D1F28;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     ecx = ebp + -80;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
@@ -2909,7 +2909,7 @@ loc_002BD4F9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D1F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2939,7 +2939,7 @@ loc_002BD4CE: ;
     g_seh_ebp = ebp; sub_0025AB90(); return; /* tail jmp 0x0025AB90 */
 
     eax = 0x3D1F28;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     ecx = ebp + -80;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
@@ -2959,7 +2959,7 @@ loc_002BD4F9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D1F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3003,7 +3003,7 @@ loc_002BD4F9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D1F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3044,7 +3044,7 @@ loc_002BD4F9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D1F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3082,7 +3082,7 @@ loc_002BD4F9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D1F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3117,7 +3117,7 @@ loc_002BD504: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D1F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3147,7 +3147,7 @@ loc_002BD506: ;
     g_seh_ebp = ebp; sub_0025A630(); return; /* tail jmp 0x0025A630 */
 
     eax = 0x3D1F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3248,7 +3248,7 @@ loc_002BD59D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D202C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3299,7 +3299,7 @@ loc_002BD59D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D202C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3347,7 +3347,7 @@ loc_002BD59D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D202C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3392,7 +3392,7 @@ loc_002BD59D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D202C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3434,7 +3434,7 @@ loc_002BD59D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D202C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3471,7 +3471,7 @@ loc_002BD59F: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D202C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6007,7 +6007,7 @@ loc_002BD788: ;
     g_seh_ebp = ebp; sub_0025AB90(); return; /* tail jmp 0x0025AB90 */
 
     eax = 0x3D2170;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6043,7 +6043,7 @@ loc_002BD790: ;
     g_seh_ebp = ebp; sub_0025AB90(); return; /* tail jmp 0x0025AB90 */
 
     eax = 0x3D2170;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6205,7 +6205,7 @@ loc_002BD7B8: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6333,7 +6333,7 @@ loc_002BD7C3: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6455,7 +6455,7 @@ loc_002BD7CE: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6571,7 +6571,7 @@ loc_002BD7D9: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6681,7 +6681,7 @@ loc_002BD7E4: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6785,7 +6785,7 @@ loc_002BD7EF: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6883,7 +6883,7 @@ loc_002BD7FA: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6975,7 +6975,7 @@ loc_002BD808: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7061,7 +7061,7 @@ loc_002BD816: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7141,7 +7141,7 @@ loc_002BD824: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7215,7 +7215,7 @@ loc_002BD832: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7283,7 +7283,7 @@ loc_002BD840: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7345,7 +7345,7 @@ loc_002BD84E: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7401,7 +7401,7 @@ loc_002BD85C: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7451,7 +7451,7 @@ loc_002BD86A: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7495,7 +7495,7 @@ loc_002BD878: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7533,7 +7533,7 @@ loc_002BD886: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7565,7 +7565,7 @@ loc_002BD894: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2224;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7633,7 +7633,7 @@ loc_002BD8B8: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2268;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7675,7 +7675,7 @@ loc_002BD8C3: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2268;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7711,7 +7711,7 @@ loc_002BD8CE: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2268;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7741,7 +7741,7 @@ loc_002BD8D9: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2268;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7789,7 +7789,7 @@ loc_002BD8F8: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2294;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7838,7 +7838,7 @@ loc_002BD918: ;
     g_seh_ebp = ebp; sub_0025DCA0(); return; /* tail jmp 0x0025DCA0 */
 
     eax = 0x3D22C0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7982,7 +7982,7 @@ loc_002BD9BB: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D23B0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8026,7 +8026,7 @@ loc_002BD9C6: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D23B0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8064,7 +8064,7 @@ loc_002BD9D1: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D23B0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8120,7 +8120,7 @@ loc_002BD9F8: ;
     g_seh_ebp = ebp; sub_0025F750(); return; /* tail jmp 0x0025F750 */
 
     eax = 0x3D23DC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8580,7 +8580,7 @@ loc_002BDB57: ;
     g_seh_ebp = ebp; sub_000ADA70(); return; /* tail jmp 0x000ADA70 */
 
     eax = 0x3D2534;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8657,7 +8657,7 @@ loc_002BDB98: ;
     g_seh_ebp = ebp; sub_0025DCA0(); return; /* tail jmp 0x0025DCA0 */
 
     eax = 0x3D258C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8693,7 +8693,7 @@ loc_002BDBA3: ;
     g_seh_ebp = ebp; sub_0025DCA0(); return; /* tail jmp 0x0025DCA0 */
 
     eax = 0x3D258C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8741,13 +8741,13 @@ loc_002BDBCB: ;
     g_seh_ebp = ebp; sub_002621B0(); return; /* tail jmp 0x002621B0 */
 
     eax = 0x3D25B8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     ecx = MEM32(ebp + -16);
     g_seh_ebp = ebp; sub_00261A90(); return; /* tail jmp 0x00261A90 */
 
     eax = 0x3D2600;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8783,7 +8783,7 @@ loc_002BDBE0: ;
     g_seh_ebp = ebp; sub_00261A90(); return; /* tail jmp 0x00261A90 */
 
     eax = 0x3D2600;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8965,7 +8965,7 @@ loc_002BDCA7: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2734;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9025,7 +9025,7 @@ loc_002BDCAF: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2734;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9079,7 +9079,7 @@ loc_002BDCBA: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2734;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9127,7 +9127,7 @@ loc_002BDCC5: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2734;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9169,7 +9169,7 @@ loc_002BDCD0: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2734;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9205,7 +9205,7 @@ loc_002BDCDB: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2734;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9235,7 +9235,7 @@ loc_002BDCE6: ;
     g_seh_ebp = ebp; sub_0025D760(); return; /* tail jmp 0x0025D760 */
 
     eax = 0x3D2734;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9301,7 +9301,7 @@ loc_002BDD28: ;
     g_seh_ebp = ebp; sub_00264300(); return; /* tail jmp 0x00264300 */
 
     eax = 0x3D2794;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9332,7 +9332,7 @@ loc_002BDD30: ;
     g_seh_ebp = ebp; sub_00264300(); return; /* tail jmp 0x00264300 */
 
     eax = 0x3D2794;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9360,7 +9360,7 @@ loc_002BDD38: ;
     g_seh_ebp = ebp; sub_00264300(); return; /* tail jmp 0x00264300 */
 
     eax = 0x3D2794;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9448,7 +9448,7 @@ loc_002BDD98: ;
     g_seh_ebp = ebp; sub_002621B0(); return; /* tail jmp 0x002621B0 */
 
     eax = 0x3D2834;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9532,7 +9532,7 @@ loc_002BDDF8: ;
     g_seh_ebp = ebp; sub_0025A630(); return; /* tail jmp 0x0025A630 */
 
     eax = 0x3D28A8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -9584,7 +9584,7 @@ loc_002BDE18: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D28D4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
