diff --git a/src/game/recomp/gen/recomp_0058.c b/src/game/recomp/gen/recomp_0058.c
index 67873d1..65eaf71 100644
--- a/src/game/recomp/gen/recomp_0058.c
+++ b/src/game/recomp/gen/recomp_0058.c
@@ -168,7 +168,7 @@ loc_002BDF68: ;
     g_seh_ebp = ebp; sub_0023B710(); return; /* tail jmp 0x0023B710 */
 
     eax = 0x3D2AC8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -250,7 +250,7 @@ loc_002BDFC8: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2B3C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -371,7 +371,7 @@ loc_002BE068: ;
     g_seh_ebp = ebp; sub_0023B740(); return; /* tail jmp 0x0023B740 */
 
     eax = 0x3D2C00;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -401,7 +401,7 @@ loc_002BE073: ;
     g_seh_ebp = ebp; sub_0023B740(); return; /* tail jmp 0x0023B740 */
 
     eax = 0x3D2C00;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -452,7 +452,7 @@ loc_002BE09B: ;
     g_seh_ebp = ebp; sub_002913A0(); return; /* tail jmp 0x002913A0 */
 
     eax = 0x3D2C34;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -479,7 +479,7 @@ loc_002BE0A6: ;
     g_seh_ebp = ebp; sub_002913A0(); return; /* tail jmp 0x002913A0 */
 
     eax = 0x3D2C34;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -571,7 +571,7 @@ loc_002BE0E8: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2C9C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -613,7 +613,7 @@ loc_002BE0F3: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2C9C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -649,7 +649,7 @@ loc_002BE0FE: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2C9C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -679,7 +679,7 @@ loc_002BE109: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2C9C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -906,7 +906,7 @@ loc_002BE1AE: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2D1C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -970,7 +970,7 @@ loc_002BE1AE: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2D1C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1014,7 +1014,7 @@ loc_002BE1B6: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2D1C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1052,7 +1052,7 @@ loc_002BE1C1: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2D1C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1143,7 +1143,7 @@ loc_002BE228: ;
     g_seh_ebp = ebp; sub_0023B740(); return; /* tail jmp 0x0023B740 */
 
     eax = 0x3D2D90;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1189,7 +1189,7 @@ loc_002BE248: ;
     g_seh_ebp = ebp; sub_0023B740(); return; /* tail jmp 0x0023B740 */
 
     eax = 0x3D2DBC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1322,7 +1322,7 @@ loc_002BE2C8: ;
     g_seh_ebp = ebp; sub_0023B710(); return; /* tail jmp 0x0023B710 */
 
     eax = 0x3D2E5C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1368,7 +1368,7 @@ loc_002BE2E8: ;
     g_seh_ebp = ebp; sub_00294DD0(); return; /* tail jmp 0x00294DD0 */
 
     eax = 0x3D2E90;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1404,7 +1404,7 @@ loc_002BE2F0: ;
     g_seh_ebp = ebp; sub_00294DD0(); return; /* tail jmp 0x00294DD0 */
 
     eax = 0x3D2E90;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1482,7 +1482,7 @@ loc_002BE338: ;
     g_seh_ebp = ebp; sub_00294DE0(); return; /* tail jmp 0x00294DE0 */
 
     eax = 0x3D2EE0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1646,7 +1646,7 @@ loc_002BE3A8: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1682,7 +1682,7 @@ loc_002BE3B3: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D2F6C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1942,7 +1942,7 @@ loc_002BE45E: ;
     g_seh_ebp = ebp; sub_00297430(); return; /* tail jmp 0x00297430 */
 
     eax = 0x3D3058;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -1982,7 +1982,7 @@ loc_002BE46C: ;
     g_seh_ebp = ebp; sub_00297430(); return; /* tail jmp 0x00297430 */
 
     eax = 0x3D3058;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2058,7 +2058,7 @@ loc_002BE498: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D309C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2100,7 +2100,7 @@ loc_002BE4A3: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D309C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2136,7 +2136,7 @@ loc_002BE4AE: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D309C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2166,7 +2166,7 @@ loc_002BE4B9: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D309C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2211,7 +2211,7 @@ loc_002BE4DB: ;
     g_seh_ebp = ebp; sub_00296550(); return; /* tail jmp 0x00296550 */
 
     eax = 0x3D30D0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2238,7 +2238,7 @@ loc_002BE4E6: ;
     g_seh_ebp = ebp; sub_00296550(); return; /* tail jmp 0x00296550 */
 
     eax = 0x3D30D0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2347,7 +2347,7 @@ loc_002BE575: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3114;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2428,7 +2428,7 @@ loc_002BE575: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3114;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2490,7 +2490,7 @@ loc_002BE575: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3114;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2532,7 +2532,7 @@ loc_002BE57D: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3114;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2834,7 +2834,7 @@ loc_002BE675: ;
     g_seh_ebp = ebp; sub_000ADAA0(); return; /* tail jmp 0x000ADAA0 */
 
     eax = 0x3D31B8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2904,7 +2904,7 @@ loc_002BE675: ;
     g_seh_ebp = ebp; sub_000ADAA0(); return; /* tail jmp 0x000ADAA0 */
 
     eax = 0x3D31B8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -2955,7 +2955,7 @@ loc_002BE675: ;
     g_seh_ebp = ebp; sub_000ADAA0(); return; /* tail jmp 0x000ADAA0 */
 
     eax = 0x3D31B8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3047,7 +3047,7 @@ loc_002BE6D7: ;
     g_seh_ebp = ebp; sub_000ADAA0(); return; /* tail jmp 0x000ADAA0 */
 
     eax = 0x3D3208;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3100,7 +3100,7 @@ loc_002BE6F8: ;
     g_seh_ebp = ebp; sub_000ADAA0(); return; /* tail jmp 0x000ADAA0 */
 
     eax = 0x3D3234;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3208,7 +3208,7 @@ loc_002BE758: ;
     g_seh_ebp = ebp; sub_00297300(); return; /* tail jmp 0x00297300 */
 
     eax = 0x3D32C0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3250,7 +3250,7 @@ loc_002BE763: ;
     g_seh_ebp = ebp; sub_00297300(); return; /* tail jmp 0x00297300 */
 
     eax = 0x3D32C0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3286,7 +3286,7 @@ loc_002BE76E: ;
     g_seh_ebp = ebp; sub_00297300(); return; /* tail jmp 0x00297300 */
 
     eax = 0x3D32C0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3316,7 +3316,7 @@ loc_002BE779: ;
     g_seh_ebp = ebp; sub_00297300(); return; /* tail jmp 0x00297300 */
 
     eax = 0x3D32C0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3415,7 +3415,7 @@ loc_002BE7DE: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3314;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3485,7 +3485,7 @@ loc_002BE7DE: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3314;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3535,7 +3535,7 @@ loc_002BE7E6: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3314;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3579,7 +3579,7 @@ loc_002BE7F1: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3314;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3614,7 +3614,7 @@ loc_002BE7FC: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3314;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3646,7 +3646,7 @@ loc_002BE804: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3314;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -3775,7 +3775,7 @@ loc_002BE8C9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -3851,7 +3851,7 @@ loc_002BE8C9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -3922,7 +3922,7 @@ loc_002BE8C9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -3979,7 +3979,7 @@ loc_002BE8C9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -4033,7 +4033,7 @@ loc_002BE8C9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -4084,7 +4084,7 @@ loc_002BE8C9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -4132,7 +4132,7 @@ loc_002BE8C9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -4177,7 +4177,7 @@ loc_002BE8C9: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -4217,7 +4217,7 @@ loc_002BE8E6: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -4240,7 +4240,7 @@ loc_002BE8EA: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D33AC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
 
@@ -4305,7 +4305,7 @@ loc_002BE908: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D33F0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4347,7 +4347,7 @@ loc_002BE913: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D33F0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4383,7 +4383,7 @@ loc_002BE91E: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D33F0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4413,7 +4413,7 @@ loc_002BE929: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D33F0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4519,7 +4519,7 @@ loc_002BE9B5: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3434;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4600,7 +4600,7 @@ loc_002BE9B5: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3434;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4662,7 +4662,7 @@ loc_002BE9B5: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3434;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4704,7 +4704,7 @@ loc_002BE9BD: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3434;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4829,7 +4829,7 @@ loc_002BEA28: ;
     g_seh_ebp = ebp; sub_00298530(); return; /* tail jmp 0x00298530 */
 
     eax = 0x3D34C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4882,7 +4882,7 @@ loc_002BEA33: ;
     g_seh_ebp = ebp; sub_00298530(); return; /* tail jmp 0x00298530 */
 
     eax = 0x3D34C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4929,7 +4929,7 @@ loc_002BEA3E: ;
     g_seh_ebp = ebp; sub_00298530(); return; /* tail jmp 0x00298530 */
 
     eax = 0x3D34C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -4970,7 +4970,7 @@ loc_002BEA49: ;
     g_seh_ebp = ebp; sub_00298530(); return; /* tail jmp 0x00298530 */
 
     eax = 0x3D34C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5005,7 +5005,7 @@ loc_002BEA54: ;
     g_seh_ebp = ebp; sub_00298530(); return; /* tail jmp 0x00298530 */
 
     eax = 0x3D34C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5125,7 +5125,7 @@ loc_002BEAE5: ;
     g_seh_ebp = ebp; sub_00298500(); return; /* tail jmp 0x00298500 */
 
     eax = 0x3D351C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5212,7 +5212,7 @@ loc_002BEAE5: ;
     g_seh_ebp = ebp; sub_00298500(); return; /* tail jmp 0x00298500 */
 
     eax = 0x3D351C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5280,7 +5280,7 @@ loc_002BEAE5: ;
     g_seh_ebp = ebp; sub_00298500(); return; /* tail jmp 0x00298500 */
 
     eax = 0x3D351C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5328,7 +5328,7 @@ loc_002BEAED: ;
     g_seh_ebp = ebp; sub_00298500(); return; /* tail jmp 0x00298500 */
 
     eax = 0x3D351C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5370,7 +5370,7 @@ loc_002BEAF8: ;
     g_seh_ebp = ebp; sub_00298500(); return; /* tail jmp 0x00298500 */
 
     eax = 0x3D351C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5403,7 +5403,7 @@ loc_002BEB03: ;
     g_seh_ebp = ebp; sub_00298500(); return; /* tail jmp 0x00298500 */
 
     eax = 0x3D351C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5501,7 +5501,7 @@ loc_002BEB72: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D3584;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5549,7 +5549,7 @@ loc_002BEB72: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D3584;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5592,7 +5592,7 @@ loc_002BEB92: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D3584;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5619,7 +5619,7 @@ loc_002BEB96: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3584;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5670,7 +5670,7 @@ loc_002BEBB8: ;
     g_seh_ebp = ebp; sub_000ADAA0(); return; /* tail jmp 0x000ADAA0 */
 
     eax = 0x3D35B0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5733,7 +5733,7 @@ loc_002BEBF7: ;
     g_seh_ebp = ebp; sub_000ADAA0(); return; /* tail jmp 0x000ADAA0 */
 
     eax = 0x3D35DC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5803,7 +5803,7 @@ loc_002BEC38: ;
     g_seh_ebp = ebp; sub_0023A4B0(); return; /* tail jmp 0x0023A4B0 */
 
     eax = 0x3D362C;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5913,7 +5913,7 @@ loc_002BECC5: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3670;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -5994,7 +5994,7 @@ loc_002BECC5: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3670;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6056,7 +6056,7 @@ loc_002BECC5: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3670;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6098,7 +6098,7 @@ loc_002BECCD: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D3670;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6176,7 +6176,7 @@ loc_002BECF8: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D36B4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6218,7 +6218,7 @@ loc_002BED03: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D36B4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6254,7 +6254,7 @@ loc_002BED0E: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D36B4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6284,7 +6284,7 @@ loc_002BED19: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D36B4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6367,7 +6367,7 @@ loc_002BED58: ;
     g_seh_ebp = ebp; sub_00297300(); return; /* tail jmp 0x00297300 */
 
     eax = 0x3D3714;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6414,7 +6414,7 @@ loc_002BED63: ;
     g_seh_ebp = ebp; sub_00297300(); return; /* tail jmp 0x00297300 */
 
     eax = 0x3D3714;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6455,7 +6455,7 @@ loc_002BED6E: ;
     g_seh_ebp = ebp; sub_00297300(); return; /* tail jmp 0x00297300 */
 
     eax = 0x3D3714;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6556,7 +6556,7 @@ loc_002BEDDE: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3758;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6620,7 +6620,7 @@ loc_002BEDDE: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3758;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6664,7 +6664,7 @@ loc_002BEDE6: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3758;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6699,7 +6699,7 @@ loc_002BEDF1: ;
     g_seh_ebp = ebp; sub_002972D0(); return; /* tail jmp 0x002972D0 */
 
     eax = 0x3D3758;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6803,7 +6803,7 @@ loc_002BEE6D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D37C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6860,7 +6860,7 @@ loc_002BEE6D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D37C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6914,7 +6914,7 @@ loc_002BEE6D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D37C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6963,7 +6963,7 @@ loc_002BEE8D: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D37C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -6995,7 +6995,7 @@ loc_002BEE91: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D37C8;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7048,7 +7048,7 @@ loc_002BEEBB: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D37FC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7075,7 +7075,7 @@ loc_002BEEC6: ;
     g_seh_ebp = ebp; sub_00263C90(); return; /* tail jmp 0x00263C90 */
 
     eax = 0x3D37FC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7736,7 +7736,7 @@ loc_002BF0F8: ;
     g_seh_ebp = ebp; sub_002A1A00(); return; /* tail jmp 0x002A1A00 */
 
     eax = 0x3D3B54;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7880,7 +7880,7 @@ loc_002BF1BB: ;
     g_seh_ebp = ebp; sub_002A0700(); return; /* tail jmp 0x002A0700 */
 
     eax = 0x3D3CC4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -7939,7 +7939,7 @@ loc_002BF1E8: ;
     g_seh_ebp = ebp; sub_002A0700(); return; /* tail jmp 0x002A0700 */
 
     eax = 0x3D3CF0;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8072,7 +8072,7 @@ loc_002BF2A8: ;
     g_seh_ebp = ebp; sub_002478C0(); return; /* tail jmp 0x002478C0 */
 
     eax = 0x3D3E18;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8118,7 +8118,7 @@ loc_002BF2C8: ;
     g_seh_ebp = ebp; sub_002A1720(); return; /* tail jmp 0x002A1720 */
 
     eax = 0x3D3E44;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8167,7 +8167,7 @@ loc_002BF2E8: ;
     g_seh_ebp = ebp; sub_002478C0(); return; /* tail jmp 0x002478C0 */
 
     eax = 0x3D3E78;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8203,7 +8203,7 @@ loc_002BF2F0: ;
     g_seh_ebp = ebp; sub_002478C0(); return; /* tail jmp 0x002478C0 */
 
     eax = 0x3D3E78;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8285,7 +8285,7 @@ loc_002BF346: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D3EC4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8333,7 +8333,7 @@ loc_002BF346: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D3EC4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8378,7 +8378,7 @@ loc_002BF346: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D3EC4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8420,7 +8420,7 @@ loc_002BF346: ;
     esp += 4; return; /* ret */
 
     eax = 0x3D3EC4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8446,7 +8446,7 @@ loc_002BF34A: ;
     g_seh_ebp = ebp; sub_002A1720(); return; /* tail jmp 0x002A1720 */
 
     eax = 0x3D3EC4;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
@@ -8621,7 +8621,7 @@ loc_002BF428: ;
     g_seh_ebp = ebp; sub_0023CB40(); return; /* tail jmp 0x0023CB40 */
 
     eax = 0x3D3FFC;
-    g_seh_ebp = ebp; sub_002A5319(); return; /* tail jmp 0x002A5319 */
+    g_seh_ebp = ebp; CxxFrameHandler(); return; /* tail jmp 0x002A5319 */
 
     /* int3: debug-trap slide byte, stepped over */
     /* int3: debug-trap slide byte, stepped over */
