diff --git a/src/game/recomp/gen/recomp_0042.c b/src/game/recomp/gen/recomp_0042.c
index a207c15..4cffc12 100644
--- a/src/game/recomp/gen/recomp_0042.c
+++ b/src/game/recomp/gen/recomp_0042.c
@@ -586,7 +586,7 @@ loc_001E91F0: ;
     PUSH32(esp, 0);
     PUSH32(esp, 0xA);
     MEM32(esi + 0x28) = eax;
-    PUSH32(esp, 0x001E920Bu); RECOMP_ABI_CALL(0x002A31D5u, sub_002A31D5); /* call 0x002A31D5 */
+    PUSH32(esp, 0x001E920Bu); RECOMP_ABI_CALL(0x002A31D5u, rand_002A31D5); /* call 0x002A31D5 */
 
 loc_001E920B: ;
     MEM32(esp + 0x1C) = eax;
@@ -1041,7 +1041,7 @@ loc_001E9514: ;
     PUSH32(esp, 0);
     PUSH32(esp, 0);
     PUSH32(esp, 0xA);
-    PUSH32(esp, 0x001E9525u); RECOMP_ABI_CALL(0x002A31D5u, sub_002A31D5); /* call 0x002A31D5 */
+    PUSH32(esp, 0x001E9525u); RECOMP_ABI_CALL(0x002A31D5u, rand_002A31D5); /* call 0x002A31D5 */
 
 loc_001E9525: ;
     MEM32(esp + 0x48) = eax;
@@ -1621,7 +1621,7 @@ loc_001E9875: ;
     PUSH32(esp, 0x3B6928);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001E988Fu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001E988Fu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001E988F: ;
     edx = MEM32(0x42BFD0);
@@ -1684,7 +1684,7 @@ loc_001E9914: ;
     PUSH32(esp, 0x3B6914);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001E992Eu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001E992Eu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001E992E: ;
     edx = MEM32(0x42BFD0);
@@ -1747,7 +1747,7 @@ loc_001E99B3: ;
     PUSH32(esp, 0x3B6904);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001E99CDu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001E99CDu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001E99CD: ;
     edx = MEM32(0x42BFD0);
@@ -1810,7 +1810,7 @@ loc_001E9A55: ;
     PUSH32(esp, 0x3B68F0);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001E9A6Fu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001E9A6Fu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001E9A6F: ;
     edx = MEM32(0x42BFD0);
@@ -1873,7 +1873,7 @@ loc_001E9AF4: ;
     PUSH32(esp, 0x3B68E4);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001E9B0Eu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001E9B0Eu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001E9B0E: ;
     edx = MEM32(0x42BFD0);
@@ -2643,7 +2643,7 @@ loc_001E9F36: ;
     PUSH32(esp, 0x3B7E34);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001E9F50u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001E9F50u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001E9F50: ;
     edx = MEM32(0x42BFD0);
@@ -2708,7 +2708,7 @@ loc_001E9FDD: ;
     PUSH32(esp, 0x3B7E20);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001E9FF7u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001E9FF7u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001E9FF7: ;
     edx = MEM32(0x42BFD0);
@@ -2772,7 +2772,7 @@ loc_001EA07F: ;
     PUSH32(esp, 0x3B7E10);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA099u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA099u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA099: ;
     edx = MEM32(0x42BFD0);
@@ -2836,7 +2836,7 @@ loc_001EA121: ;
     PUSH32(esp, 0x3B7E00);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA13Bu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA13Bu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA13B: ;
     edx = MEM32(0x42BFD0);
@@ -2900,7 +2900,7 @@ loc_001EA1C3: ;
     PUSH32(esp, 0x3B7DEC);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA1DDu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA1DDu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA1DD: ;
     edx = MEM32(0x42BFD0);
@@ -2964,7 +2964,7 @@ loc_001EA265: ;
     PUSH32(esp, 0x3B7DD8);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA27Fu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA27Fu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA27F: ;
     edx = MEM32(0x42BFD0);
@@ -3028,7 +3028,7 @@ loc_001EA307: ;
     PUSH32(esp, 0x3B7DC4);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA321u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA321u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA321: ;
     edx = MEM32(0x42BFD0);
@@ -3092,7 +3092,7 @@ loc_001EA3A9: ;
     PUSH32(esp, 0x3B7DB0);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA3C3u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA3C3u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA3C3: ;
     edx = MEM32(0x42BFD0);
@@ -3156,7 +3156,7 @@ loc_001EA44B: ;
     PUSH32(esp, 0x3B7D9C);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA465u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA465u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA465: ;
     edx = MEM32(0x42BFD0);
@@ -3220,7 +3220,7 @@ loc_001EA4ED: ;
     PUSH32(esp, 0x3B7D84);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA507u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA507u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA507: ;
     edx = MEM32(0x42BFD0);
@@ -3284,7 +3284,7 @@ loc_001EA58F: ;
     PUSH32(esp, 0x3B7D70);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA5A9u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA5A9u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA5A9: ;
     edx = MEM32(0x42BFD0);
@@ -3348,7 +3348,7 @@ loc_001EA631: ;
     PUSH32(esp, 0x3B7D5C);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA64Bu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA64Bu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA64B: ;
     edx = MEM32(0x42BFD0);
@@ -3412,7 +3412,7 @@ loc_001EA6D3: ;
     PUSH32(esp, 0x3B7D4C);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA6EDu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA6EDu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA6ED: ;
     edx = MEM32(0x42BFD0);
@@ -3476,7 +3476,7 @@ loc_001EA775: ;
     PUSH32(esp, 0x3B7D3C);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA78Fu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA78Fu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA78F: ;
     edx = MEM32(0x42BFD0);
@@ -3540,7 +3540,7 @@ loc_001EA817: ;
     PUSH32(esp, 0x3B7D2C);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA831u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA831u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA831: ;
     edx = MEM32(0x42BFD0);
@@ -3605,7 +3605,7 @@ loc_001EA8BC: ;
     PUSH32(esp, 0x3B7D1C);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA8D6u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA8D6u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA8D6: ;
     edx = MEM32(0x42BFD0);
@@ -3670,7 +3670,7 @@ loc_001EA961: ;
     PUSH32(esp, 0x3B7D0C);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EA97Bu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EA97Bu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EA97B: ;
     edx = MEM32(0x42BFD0);
@@ -3735,7 +3735,7 @@ loc_001EAA0A: ;
     PUSH32(esp, 0x3B7CFC);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EAA24u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EAA24u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EAA24: ;
     edx = MEM32(0x42BFD0);
@@ -3800,7 +3800,7 @@ loc_001EAAB3: ;
     PUSH32(esp, 0x3B7CEC);
     PUSH32(esp, eax);
     MEM32(ecx * 4 + 0x42B3D0) = eax;
-    PUSH32(esp, 0x001EAACDu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EAACDu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EAACD: ;
     edx = MEM32(0x42BFD0);
@@ -11160,7 +11160,7 @@ loc_001ED62E: ;
 loc_001ED630: ;
     PUSH32(esp, 0x3B7FEC);
     PUSH32(esp, esi);
-    PUSH32(esp, 0x001ED63Bu); RECOMP_ABI_CALL(0x002A4C7Du, sub_002A4C7D); /* call 0x002A4C7D */
+    PUSH32(esp, 0x001ED63Bu); RECOMP_ABI_CALL(0x002A4C7Du, stricmp_002A4C7D); /* call 0x002A4C7D */
 
 loc_001ED63B: ;
     _fb = (uint32_t)(8) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -11180,7 +11180,7 @@ loc_001ED642: ;
 loc_001ED645: ;
     PUSH32(esp, 0x3B7FD8);
     PUSH32(esp, esi);
-    PUSH32(esp, 0x001ED650u); RECOMP_ABI_CALL(0x002A4C7Du, sub_002A4C7D); /* call 0x002A4C7D */
+    PUSH32(esp, 0x001ED650u); RECOMP_ABI_CALL(0x002A4C7Du, stricmp_002A4C7D); /* call 0x002A4C7D */
 
 loc_001ED650: ;
     _fb = (uint32_t)(8) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -11201,7 +11201,7 @@ loc_001ED657: ;
 loc_001ED65F: ;
     PUSH32(esp, 0x3B7FC4);
     PUSH32(esp, esi);
-    PUSH32(esp, 0x001ED66Au); RECOMP_ABI_CALL(0x002A4C7Du, sub_002A4C7D); /* call 0x002A4C7D */
+    PUSH32(esp, 0x001ED66Au); RECOMP_ABI_CALL(0x002A4C7Du, stricmp_002A4C7D); /* call 0x002A4C7D */
 
 loc_001ED66A: ;
     _fb = (uint32_t)(8) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -11222,7 +11222,7 @@ loc_001ED671: ;
 loc_001ED679: ;
     PUSH32(esp, 0x3B7FB8);
     PUSH32(esp, esi);
-    PUSH32(esp, 0x001ED684u); RECOMP_ABI_CALL(0x002A4C7Du, sub_002A4C7D); /* call 0x002A4C7D */
+    PUSH32(esp, 0x001ED684u); RECOMP_ABI_CALL(0x002A4C7Du, stricmp_002A4C7D); /* call 0x002A4C7D */
 
 loc_001ED684: ;
     _fb = (uint32_t)(8) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -13081,7 +13081,7 @@ loc_001EDF95: ;
     PUSH32(esp, ecx);
     MEM32(esi + 8) = edx;
     MEM32(esi + 0x38) = 0;
-    PUSH32(esp, 0x001EDFBFu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EDFBFu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EDFBF: ;
     edx = MEM32(esi + 0xC);
@@ -13248,7 +13248,7 @@ loc_001EE070: ;
     PUSH32(esp, ebp);
     PUSH32(esp, ebx);
     PUSH32(esp, ecx);
-    PUSH32(esp, 0x001EE07Eu); RECOMP_ABI_CALL(0x002A4DAEu, sub_002A4DAE); /* call 0x002A4DAE */
+    PUSH32(esp, 0x001EE07Eu); RECOMP_ABI_CALL(0x002A4DAEu, strnicmp_002A4DAE); /* call 0x002A4DAE */
 
 loc_001EE07E: ;
     _fb = (uint32_t)(0xC) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -13722,7 +13722,7 @@ loc_001EE2BB: ;
     PUSH32(esp, eax);
     PUSH32(esp, edi);
     PUSH32(esp, ebp);
-    PUSH32(esp, 0x001EE2C5u); RECOMP_ABI_CALL(0x002A3970u, sub_002A3970); /* call 0x002A3970 */
+    PUSH32(esp, 0x001EE2C5u); RECOMP_ABI_CALL(0x002A3970u, strncmp_002A3970); /* call 0x002A3970 */
 
 loc_001EE2C5: ;
     _fb = (uint32_t)(0xC) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -13738,7 +13738,7 @@ loc_001EE2CC: ;
     PUSH32(esp, eax);
     PUSH32(esp, edi);
     PUSH32(esp, ecx);
-    PUSH32(esp, 0x001EE2DAu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001EE2DAu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001EE2DA: ;
     edx = MEM32(ebx + 0x24);
@@ -19459,7 +19459,7 @@ loc_001F0A90: ;
     _fa = (uint32_t)(ecx) & 0xFFFFFFFFu; _fas = (int32_t)(int32_t)(_fa); /* add result */
     PUSH32(esp, ebx);
     PUSH32(esp, ecx);
-    PUSH32(esp, 0x001F0AA6u); RECOMP_ABI_CALL(0x002A4C7Du, sub_002A4C7D); /* call 0x002A4C7D */
+    PUSH32(esp, 0x001F0AA6u); RECOMP_ABI_CALL(0x002A4C7Du, stricmp_002A4C7D); /* call 0x002A4C7D */
 
 loc_001F0AA6: ;
     _fb = (uint32_t)(8) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -19961,7 +19961,7 @@ loc_001F0C9D: ;
     _fa = (uint32_t)(eax) & 0xFFFFFFFFu; _fas = (int32_t)(int32_t)(_fa); /* add result */
     PUSH32(esp, ebp);
     PUSH32(esp, eax);
-    PUSH32(esp, 0x001F0CA7u); RECOMP_ABI_CALL(0x002A4C7Du, sub_002A4C7D); /* call 0x002A4C7D */
+    PUSH32(esp, 0x001F0CA7u); RECOMP_ABI_CALL(0x002A4C7Du, stricmp_002A4C7D); /* call 0x002A4C7D */
 
 loc_001F0CA7: ;
     _fb = (uint32_t)(8) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -20521,7 +20521,7 @@ loc_001F0F20: ;
     PUSH32(esp, eax);
     eax = esp + 8;
     PUSH32(esp, eax);
-    PUSH32(esp, 0x001F0F2Du); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001F0F2Du); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001F0F2D: ;
     ecx = esp + 0xC;
@@ -20560,7 +20560,7 @@ loc_001F0F62: ;
 
 loc_001F0F6A: ;
     ecx = MEM32(esp + 0x28);
-    PUSH32(esp, 0x001F0F73u); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F0F73u); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F0F73: ;
     _fb = (uint32_t)(0x2C) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -23988,7 +23988,7 @@ loc_001F24D0: ;
     _fb = (uint32_t)(0x49E758) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
     ebp = ebp + 0x49E758;
     _fa = (uint32_t)(ebp) & 0xFFFFFFFFu; _fas = (int32_t)(int32_t)(_fa); /* add result */
-    PUSH32(esp, 0x001F24FFu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001F24FFu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001F24FF: ;
     edx = esp + 0x28;
@@ -24010,7 +24010,7 @@ loc_001F2518: ;
     eax = esp + 0x20;
     PUSH32(esp, eax);
     PUSH32(esp, ebp);
-    PUSH32(esp, 0x001F2525u); RECOMP_ABI_CALL(0x002A4DAEu, sub_002A4DAE); /* call 0x002A4DAE */
+    PUSH32(esp, 0x001F2525u); RECOMP_ABI_CALL(0x002A4DAEu, strnicmp_002A4DAE); /* call 0x002A4DAE */
 
 loc_001F2525: ;
     _fb = (uint32_t)(0xC) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -24025,7 +24025,7 @@ loc_001F252C: ;
     eax = 1;
     POP32(esp, ebx);
     ecx = MEM32(esp + 0x34);
-    PUSH32(esp, 0x001F253Cu); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F253Cu); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F253C: ;
     _fb = (uint32_t)(0x38) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -24048,7 +24048,7 @@ loc_001F2549: ;
     ecx = esp + 0x24;
     PUSH32(esp, ecx);
     PUSH32(esp, ebp);
-    PUSH32(esp, 0x001F2557u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001F2557u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001F2557: ;
     PUSH32(esp, 2);
@@ -24073,7 +24073,7 @@ loc_001F257A: ;
     POP32(esp, ebp);
     POP32(esp, ebx);
     ecx = MEM32(esp + 0x34);
-    PUSH32(esp, 0x001F2586u); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F2586u); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F2586: ;
     _fb = (uint32_t)(0x38) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -24165,7 +24165,7 @@ loc_001F25FC: ;
     _fa = (uint32_t)(eax) & 0xFFFFFFFFu; _fas = (int32_t)(int32_t)(_fa); /* xor result */
     POP32(esp, ebx);
     ecx = MEM32(esp + 0x34);
-    PUSH32(esp, 0x001F260Eu); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F260Eu); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F260E: ;
     _fb = (uint32_t)(0x38) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -24308,7 +24308,7 @@ loc_001F26F9: ;
     POP32(esp, ebp);
     eax = 1;
     POP32(esp, ebx);
-    PUSH32(esp, 0x001F270Eu); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F270Eu); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F270E: ;
     _fb = (uint32_t)(0x38) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -24584,7 +24584,7 @@ loc_001F286A: ;
     edx = esi + 0x8C;
     PUSH32(esp, 0x3B81FC);
     PUSH32(esp, edx);
-    PUSH32(esp, 0x001F287Cu); RECOMP_ABI_CALL(0x002A3340u, sub_002A3340); /* call 0x002A3340 */
+    PUSH32(esp, 0x001F287Cu); RECOMP_ABI_CALL(0x002A3340u, sprintf_002A3340); /* call 0x002A3340 */
 
 loc_001F287C: ;
     _fb = (uint32_t)(0xC) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -25074,7 +25074,7 @@ loc_001F2B50: ;
     PUSH32(esp, eax);
     ecx = esp + 0x64;
     PUSH32(esp, ecx);
-    PUSH32(esp, 0x001F2B79u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001F2B79u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001F2B79: ;
     edx = esp + 0x68;
@@ -25132,7 +25132,7 @@ loc_001F2BC8: ;
 loc_001F2BCD: ;
     eax = ecx;
     ecx = MEM32(esp + 0xDC);
-    PUSH32(esp, 0x001F2BDBu); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F2BDBu); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F2BDB: ;
     _fb = (uint32_t)(0xE0) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -25183,7 +25183,7 @@ loc_001F2C1C: ;
 loc_001F2C25: ;
     POP32(esp, edi);
     ecx = MEM32(esp + 0xDC);
-    PUSH32(esp, 0x001F2C32u); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F2C32u); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F2C32: ;
     _fb = (uint32_t)(0xE0) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -25240,7 +25240,7 @@ loc_001F2C73: ;
     PUSH32(esp, eax);
     ebp = esi + 0x8C;
     PUSH32(esp, ebp);
-    PUSH32(esp, 0x001F2C86u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001F2C86u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001F2C86: ;
     ebx = MEM32(esp + 0x104);
@@ -25280,7 +25280,7 @@ loc_001F2CBC: ;
     _fa = (uint32_t)(eax) & 0xFFFFFFFFu; _fas = (int32_t)(int32_t)(_fa); /* xor result */
     POP32(esp, edi);
     ecx = MEM32(esp + 0xDC);
-    PUSH32(esp, 0x001F2CD1u); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F2CD1u); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F2CD1: ;
     _fb = (uint32_t)(0xE0) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -25382,7 +25382,7 @@ loc_001F2D58: ;
     POP32(esp, ebp);
     POP32(esp, ebx);
     POP32(esp, edi);
-    PUSH32(esp, 0x001F2D6Au); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F2D6Au); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F2D6A: ;
     _fb = (uint32_t)(0xE0) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -25985,7 +25985,7 @@ loc_001F30E0: ;
     PUSH32(esp, eax);
     eax = esp + 8;
     PUSH32(esp, eax);
-    PUSH32(esp, 0x001F30EDu); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001F30EDu); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001F30ED: ;
     ecx = esp + 0xC;
@@ -26024,7 +26024,7 @@ loc_001F3122: ;
 
 loc_001F312A: ;
     ecx = MEM32(esp + 0x28);
-    PUSH32(esp, 0x001F3133u); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F3133u); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F3133: ;
     _fb = (uint32_t)(0x2C) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -26080,7 +26080,7 @@ loc_001F316A: ;
     eax = 1;
     POP32(esp, ebx);
     ecx = MEM32(esp + 0x298);
-    PUSH32(esp, 0x001F317Du); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F317Du); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F317D: ;
     _fb = (uint32_t)(0x29C) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -26099,7 +26099,7 @@ loc_001F318B: ;
     _fa = (uint32_t)(eax) & 0xFFFFFFFFu; _fas = (int32_t)(int32_t)(_fa); /* xor result */
     POP32(esp, ebx);
     ecx = MEM32(esp + 0x298);
-    PUSH32(esp, 0x001F319Bu); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F319Bu); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F319B: ;
     _fb = (uint32_t)(0x29C) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -26112,7 +26112,7 @@ loc_001F31A2: ;
     eax = esp + 0x24;
     PUSH32(esp, ebx);
     PUSH32(esp, eax);
-    PUSH32(esp, 0x001F31B2u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001F31B2u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001F31B2: ;
     ecx = esp + 0x2C;
@@ -26168,7 +26168,7 @@ loc_001F320B: ;
     POP32(esp, edi);
     POP32(esp, ebx);
     ecx = MEM32(esp + 0x298);
-    PUSH32(esp, 0x001F321Au); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F321Au); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F321A: ;
     _fb = (uint32_t)(0x29C) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -26800,7 +26800,7 @@ loc_001F36AC: ;
     eax = 1;
     POP32(esp, ebx);
     ecx = MEM32(esp + 0x298);
-    PUSH32(esp, 0x001F36C1u); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F36C1u); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F36C1: ;
     _fb = (uint32_t)(0x29C) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -26828,7 +26828,7 @@ loc_001F36DC: ;
     eax = 0; /* xor self */
     _fa = (uint32_t)(eax) & 0xFFFFFFFFu; _fas = (int32_t)(int32_t)(_fa); /* xor result */
     POP32(esp, ebx);
-    PUSH32(esp, 0x001F36F1u); RECOMP_ABI_CALL(0x002A5282u, sub_002A5282); /* call 0x002A5282 */
+    PUSH32(esp, 0x001F36F1u); RECOMP_ABI_CALL(0x002A5282u, security_check_cookie); /* call 0x002A5282 */
 
 loc_001F36F1: ;
     _fb = (uint32_t)(0x29C) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
@@ -27704,7 +27704,7 @@ loc_001F3B71: ;
     if (TEST_Z(_fa, _fb)) goto loc_001F3BA4; /* je: equal / zero */
 
 loc_001F3B7B: ;
-    PUSH32(esp, 0x001F3B80u); RECOMP_ABI_CALL(0x002A31D5u, sub_002A31D5); /* call 0x002A31D5 */
+    PUSH32(esp, 0x001F3B80u); RECOMP_ABI_CALL(0x002A31D5u, rand_002A31D5); /* call 0x002A31D5 */
 
 loc_001F3B80: ;
     MEM32(esp + 0x10) = eax;
@@ -28687,7 +28687,7 @@ loc_001F40D2: ;
     PUSH32(esp, esi);
     MEM32(0x49E834) = ecx;
     MEM32(0x49E830) = eax;
-    PUSH32(esp, 0x001F4107u); RECOMP_ABI_CALL(0x002A3710u, sub_002A3710); /* call 0x002A3710 */
+    PUSH32(esp, 0x001F4107u); RECOMP_ABI_CALL(0x002A3710u, strncpy_002A3710); /* call 0x002A3710 */
 
 loc_001F4107: ;
     ecx = MEM32(0x49E838);
