diff --git a/src/game/recomp/gen/recomp_0053.c b/src/game/recomp/gen/recomp_0053.c
index b793651..13bf0ed 100644
--- a/src/game/recomp/gen/recomp_0053.c
+++ b/src/game/recomp/gen/recomp_0053.c
@@ -21556,7 +21556,7 @@ loc_0029E662: ;
     edx = esi + 0xC;
     PUSH32(esp, edi);
     PUSH32(esp, edx);
-    PUSH32(esp, 0x0029E66Eu); RECOMP_ABI_CALL(0x0023CA00u, sub_0023CA00); /* call 0x0023CA00 */
+    PUSH32(esp, 0x0029E66Eu); RECOMP_ABI_CALL(0x0023CA00u, strncpy_0023CA00); /* call 0x0023CA00 */
 
 loc_0029E66E: ;
     _fb = (uint32_t)(0xC) & 0xFFFFFFFFu; _fbs = (int32_t)(int32_t)(_fb); /* add source, before the write */
